styleguide/templates/styleguide/styleguide.html

```diff
@@ -21,8 +21,8 @@
             <div class="devices">
                 <img class="devices" src ="{% static "img/devices4.png" %}">
             </div>
-            <div class="search">
-                <img class="search" src ="{% static "img/mag-glass2.png" %}">
+            <div class="app-search">
+                <img class="search-img" src ="{% static "img/mag-glass2.png" %}">
```

Comment:
Where do these classnames even come from and should we even keep this search input that does nothing?

Reply:
The whole "tool bar" is just gibberish.  We need to rip out eventually, but for now it looks pretty and that's important for morale.  I don't think we need to worry about naming conventions for something that needs to get ripped out/changed.

Reply:
+1